1-org/modules/centralized-logging/main.tf

```
@@ -124,6 +138,20 @@ resource "google_project_iam_member" "logbucket_sink_member" {
   member = module.log_export["${each.value}_lbk"].writer_identity
 }
 
+#------------------------------------------------------------------#
+# Log Bucket Service account IAM membership for log_export_billing #
+#------------------------------------------------------------------#
+resource "google_project_iam_member" "logbucket_sink_member_billing" {
+  for_each = var.logbucket_options != null ? local.logbucket_sink_member : {}
```

In order to use `module.log_export_billing[..]`, we need to ensure that `var.enable_billing_account_sink` is true, otherwise the module won't be available to be used, and this resource will raise an error.

Maybe something like should do the trick:
```suggestion
  for_each = var.enable_billing_account_sink == true and var.logbucket_options != null ? local.logbucket_sink_member : {}
```